model/database.go

```diff
@@ -0,0 +1,22 @@
+package model
+
+import (
+	"log"
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+)
+
+func Database() (*gorm.DB, error) {
+
+	db, err := gorm.Open(mysql.Open("./database.db"), &gorm.Config{})
+
+	if err != nil {
+		log.Fatal(err.Error())
```

Comment:
Lets short-circuit and return the err if present:
```suggestion
		return nil, fmt.Errorf("Database: failed connecting to db: %w", `err)
```